delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -1074,8 +1084,10 @@ def loadServiceInfo() {
 			def token = sh (script: "curl --silent -X POST -k -v \
 				-H \"Content-Type: application/json\" \
 				 $url \
-				-d \'${payload}\' | $jq -r '.data.token'", returnStdout:true)
-			token = token.replaceAll("[\n\r]", "")
+				-d \'${payload}\'", returnStdout:true)
+			def jsonParser = new groovy.json.JsonSlurper()
+			def resultJson = jsonParser.parseText(token)
+			token = resultJson.data.token.replaceAll("[\n\r]", "")
```

Comment:
check if resultJson.data is null or not before doing replaceAll

Reply:
Code Changed as suggested

Reply:
I'm not able to see the change here ... am I missing something?

Reply:
suggested changes has been made